### DISCUSSION TITLE
디시인사이드에서 개념글에서 특정 추천 수 이하의 글을 지우는 사용자 규칙을 짤 수 있을까요?

### DISCUSSION THREAD

**Discussion by 3k2jd0j2k** [List-KR/community#55] [Q&A]
가능하다면 각 갤러리 개념글에서 vpn써가면서 주작기 돌려서 최소 추천 컷만 맞춘 낚시 글들을 아예 안보고 싶습니다

https://gall.dcinside.com/board/lists/?id=comic_new4&exception_mode=recommend
예를 들어서 만화갤러리 개념글페이지에 가면 만화갤러리에서 주작충이 항상 15~20 추천 정도를 주작기 돌려서 가짜 낚시글을 개념글에 보냅니다
이런 주소로 들어갔을때 추천 수 20 이하의 글을 지워버리는 식으로 사용자 규칙을 짜는게 가능할까요?

A.
직접 몇일간 삽질해본결과 사용자규칙으론 원하는 기능의 구현이 힘들어서 유저스크립트로 대충 짜서 해결봤습니다.

**Comment by piquark6046** [comment]
모바일에서요?
